db/migrations/20230104053820_create_order_products_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE order_products (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  quantity INT NOT NULL,
+  products_id INT NOT NULL,
+  orders_id INT NOT NULL,
```

Comment:
```suggestion
  product_id INT NOT NULL,
  order_id INT NOT NULL,
```
복수, 단수 컨벤션 지켜주세요. 다른 테이블도 마찬가지입니다.